### ISSUE TITLE
File missing

### ISSUE THREAD

**Issue by SharhadBashar** [zengxianyu/jsws#7]
This file is missing in self_seg_full_sal_train.py line 52:
`voc_train_split = './trainself.txt'`

is this the same as trainaug.txt used in weak_seg_full_sal_train.py?

**Comment by zengxianyu** [comment]
It is not the same as trainaug.txt. ```./trainself.txt"``` is obtained by running ```syn_stage1.py```

**Comment by jieruyao49** [comment]
> It is not the same as trainaug.txt. `./trainself.txt"` is obtained by running `syn_stage1.py`

when i run `syn_stage1.py`, i didn't get `./trainself.txt`,how should i do?

**Comment by zengxianyu** [comment]
you can generate the file according to the format of trainaug.txt but with the file names replaced by those produced by syn_stage1.py



---Original---

**Comment by vtqtuan** [comment]
Can you provide trained models for us to test demo pictures?

**Comment by zengxianyu** [comment]
No. The trained model file was lost. You may train a model using the provided training script. I might re-train a model later and upload new trained file
